src/networking.c

```diff
@@ -2268,6 +2271,21 @@ int processInlineBuffer(client *c) {
         c->argv_len_sum += sdslen(argv[j]);
     }
     zfree(argv);
+
+    /* Per-slot network bytes-in calculation.
+     *
+     * Within networking.c, we calculate and store the current command's ingress bytes
```

Comment:
```suggestion
     * We calculate and store the current command's ingress bytes
```
We're in networking.c, so unclear why you add that caveat.